docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,108 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge link="https://strapi.io/pricing"/> <SilverBadge link="https://strapi.io/pricing"/> <GoldBadge link="https://strapi.io/pricing" withLinkIcon/>
+
+::: warning 🚧 This API is considered unstable for now.
+<br>
+:::
+
+Role-Based Access Control (RBAC) is an approach to restricting access to some users. In a Strapi application, users of the admin panel are administrators. Their roles and permissions are [configured in the admin panel](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md). In addition to the 3 default roles (Author, Editor, and Super Admin), it's possible to create custom conditions to match specific needs of roles and permissions management. This requires an Enterprise Edition with at minimum a Bronze licence plan.
+
+#### Defining a new condition
+
+Conditions are defined as an array of objects. Each condition object can have 4 possible properties:
+
+- `displayName` (string): the condition name as showed in the admin panel
+- `name` (string): the condition name, kebab-cased
+- `plugin` (string, optional): the plugin name, kebab-cased, if your condition is based on a plugin (e.g `content-manager`)
+- `handler` (function): a function to handle the user and possibly return a `condition object` (see [using the handler function](#using-the-handler-function))
+
+Declare and register conditions in your [`bootstrap`](/developer-docs/latest/setup-deployment-guides/configurations.md#bootstrap) file located at `./config/functions/bootstrap.js` (see [Adding conditions](#adding-conditions)).
+
+#### Using the handler function
+
+The condition `handler` is a synchronous or asynchronous function that:
+
+* receives the authenticated user making the request,
+* and returns `true`, `false`, or another condition object.
+
+Returning `true` means the condition will always match:
+
+```js
+  const condition = {
+    displayName: "Can drink",
+    name: "can-drink",
+    async handler(user) {
+      if (user.age > 21) return true;
```

Comment:
Just a thought, 21 seems USA centered to me. Can we find a more inclusive example?
Also, the user is the admin user that doesn't have the `age` attribute. We doesn't allow this model to be customized in the `content-type-builder`.
The attributes you can use seem to me to be: `firstname`, `lastname`, `username` and `email`.

Reply:
:+1: to both sides, lets do something like an "email contains" `@strapi.io` or something like that.

Reply:
Thanks!
Would [this](https://github.com/strapi/documentation/pull/187/commits/0da0e7f8916364e197742710789cc1e41bc9ab49) be OK or do I need to use an example that supports IE / pre ES-2015 as well (`indexOf !== -1`)?